views/apartments/show.erb

```diff
@@ -1 +1,24 @@
 This is info about apartment #3.
+<h2>
+  <%= @apartment %>
```

Comment:
`@apartment` is an object. If you print it out to the screen, you're probably going to get something goofy like `<Apartment::ActiveRecord::Base blah blah blah>`. I think you probably wanted `@apartment.address`?